packages/core/src/components/resize-sensor/resize-sensor.md

```diff
@@ -0,0 +1,27 @@
+---
+tag: new
+---
+
+@# Resize sensor
+
+`ResizeSensor` is a higher-order component that effectively provides a
```

Comment:
nit: this is a little wordy and im not sure it's even correct. Maybe rephrase:

    `ResizeSensor` is a component that provides a `"resize"` event for its single child.

But even then it's not "providing an event" in the sense of `.addEventListener`.